Makefile

```diff
@@ -23,4 +23,4 @@ lint-fix:
 db:
 	docker-compose up -d
 clean:
-	docker-compose down
\ No newline at end of file
+	docker-compose down
```

Comment:
oh gods whose editor doesn't understand how unix lines work. I better not be vscode

Reply:
I don't think vscode is the culprit - it's actually pretty good at picking up Editorconfig settings